src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -280,6 +314,30 @@ public ColibriConferenceIQ createColibriChannels(
                         (ColibriConferenceIQ) response, contents);
 
         }
+        catch (Exception e)
+        {
+            try
+            {
+                synchronized (syncRoot)
```

Comment:
Do you think it would be possible to minimize the synchronization by moving the `synchronized (syncRoot)` inside `checkIfDisposed` and the `synchronized (stateEstimationSync)` in `trackVideoStreamsRemoved`? I don't see an obvious deadlock here, but we are posting a message to the event admin while holding two locks which may be a little risky?